lib/adminware/config.rb

```diff
@@ -0,0 +1,64 @@
+require 'yaml'
+require 'fileutils'
+
+module Adminware
+  class Config
+    #Set default values for config settings
+    DEFAULT_JOBDIR = "jobs/"
+    DEFAULT_LOGFILE = "logs/adminware.log"
+    DEFAULT_STATEDIR = "var/state.yaml"
+
+    #Load the config file
+    def initialize
+      @path = Adminware.root 
+      configfile = File.join(@path, 'etc/config.yaml')
+      @config = YAML.load_file(configfile)
+
+      #If no path given in config then set them to their defaults
+      @jobdir = @config['centraljobdir'] ||= DEFAULT_JOBDIR
+      @jobdir = set_path(@jobdir)
```

Comment:
This could be simplified to:

```
@jobdir = set_path(@config['centraljobdir'] || DEFAULT_JOBDIR)
```

I would also rename `set_path` to something a bit more descriptive.  `resolve_path`?